client/components/ChatInput.vue

```diff
@@ -210,6 +210,23 @@ export default {
 				const args = text.substr(1).split(" ");
 				const cmd = args.shift().toLowerCase();
 
+				if (cmd === "join") {
```

Comment:
Can you move this to `/js/commands/join.js` file that specifically handles the join command?

Reply:
See if you like that better. It feels a little wonky to me, but it does work.